opentelemetry-jaeger-propagator/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "opentelemetry-jaeger-propagator"
+version = "0.1.0"
+description = "Jaeger propagator for OpenTelemetry"
+homepage = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-jaeger-propagator"
+repository = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-jaeger-propagator"
+readme = "README.md"
+categories = [
+    "development-tools::debugging",
+    "development-tools::profiling",
+    "asynchronous",
+]
+keywords = ["opentelemetry", "jaeger", "propagator"]
+license = "Apache-2.0"
+edition = "2021"
+rust-version = "1.65"
+
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+
+[dependencies]
+opentelemetry = { version = "0.21", default-features = false, features = [
+    "trace",
+], path = "../opentelemetry" }
+
+tonic = { workspace = true, optional = true }
+prost = { version = "0.11.6", optional = true }
+prost-types = { version = "0.11.6", optional = true }
+
+[dev-dependencies]
+opentelemetry_sdk = { features = ["testing"], path = "../opentelemetry-sdk" }
+
+[features]
+full = []
+default = []
```

Comment:
remove those two and add `integration_test` feature

Reply:
I replaced the full and default features with integration_test which is also empty right now. I kept the integration tests in opentelemetry-jaeger instead of migrating them over so I don't have any integration tests in opentelemetry-jaeger-propagator. Does this make sense and is this what you expect?